src/packages/tsconfig.json

```diff
@@ -1,26 +0,0 @@
-{
-    "extends": "../../settings.tsconfig.json",
-    "include": ["*"],
-    "compilerOptions": {
-        "noEmit": false,
-        "module": "commonjs",
-        "target": "ES2019",
-        "allowJs": true,
-        "paths": { "@packages/*": ["./*/"] },
-        "preserveSymlinks": true
-    },
-    "references": [
-        { "path": "../server" },
```

Comment:
Keep in mind that when I mention 'get code references' I mean opening vscode fresh and finding references right away on the target file.
vs having opened several files in different projects already, which 'helps' vscode see the references in some situations.

Reply:
> I was not able to get code references working without it.

We should figure out why that is, rather than adding references that don't reflect your actual dependencies.

> finding references right away

There are a few seconds during initial project load when you will see approximate results.  We can certainly investigate whether that's lasting too long for you, but you can't count on getting all code references until the "Initializing JS/TS" spinner in the status bar goes away.

> vs having opened several files in different projects already

Yes, which files are open matters.  Thanks for confirming that your repro is with exactly the one file.  I'll keep trying to make it happen on my box.